spec/requests/stories_index_spec.rb

```diff
@@ -104,6 +104,14 @@
       expect(response.body).to include(CGI.escapeHTML(listing.title))
     end
 
+    it "does not set surrogate headers if private" do
+      allow(SiteConfig).to receive(:public).and_return(false)
+      get "/"
+      expect(response.status).to eq(200)
+
+      expect(response.headers["Surrogate-Key"]).to eq(nil)
+    end
+
```

Comment:
I believe this should check `Cache-Control` and `X-Accel-Expires` headers instead of or in addition to `Surrogate-Key`. The surrogate keys are only used for busting on Fastly. The `Cache-Control` and `X-Accel-Expires` headers are used by Fastly and Nginx (respectively) to dictate what's cached or not.

Reply:
Yep, can confirm; I think if just add an assertion to check that both `response.headers["X-Accel-Expires"]` and ``response.headers["Cache-Control"]` are `nil`, we should be good to go here 👍